src/Cli/dotnet/ToolManifest/ToolManifestFinder.cs

```diff
@@ -250,8 +250,7 @@ private string WriteManifestFile(DirectoryPath folderPath)
               "tools": {}
             }
             """;
-        _fileSystem.Directory.CreateDirectory(Path.Combine(folderPath.Value, Constants.DotConfigDirectoryName));
-        string manifestFileLocation = Path.Combine(folderPath.Value, Constants.DotConfigDirectoryName, Constants.ToolManifestFileName);
```

Comment:
Are there any back-compat concerns, where older version of the SDK installed under `.config`?